randovania/games/samus_returns/json_data/Area 5 - Entrance.txt

```diff
@@ -102,11 +99,7 @@ Extra - asset_id: collision_camera_002
   * Extra - actor_name: Door015
   * Extra - actor_type: doorpowerclosed
   > Elevator to Area 5 - Interior
-      Any of the following:
-          Phase Drift or Space Jump or Shoot Beam Burst
-          All of the following:
-              Morph Ball
-              Power Bomb or Spider Ball
+      Phase Drift or Space Jump or Lay Power Bomb or Shoot Beam Burst or Use Spider Ball
```

Comment:
Out of curiosity, whats the "shoot beam burst" used for? dont have the game memorized by map, but why not the blob thrower template?

Reply:
I should've added a comment (comments are missing everywhere) but its for killing the Autrack. After its gone you can safely jump to the ledge, and climb up.